src/style.css

```diff
@@ -1 +1,34 @@
 /* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  height: 20px;
```

Comment:
Redundant
```suggestion
```

Reply:
This is for the vertical gap 4px
<img width="425" alt="image" src="https://github.com/user-attachments/assets/6d1bb051-c13e-428e-9623-60ef3f7697c9">